maven-spy/pom.xml

```diff
@@ -46,7 +46,6 @@
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-core</artifactId>
-      <version>3.3.9</version>
```

Comment:
version definition moved to parent POM ok, but no dependency management, so where does the version come from now ? And is it still version 3.3.9 ?

(I know, old version, but changing that is irrelevant here)

Reply:
As far I can see it's here https://github.com/olamy/pipeline-maven-plugin/blob/d694dc5b789995134ac5fe6903868b867f344a1d/pom.xml#L138 (dptMngt from top pom)
this will align all maven artifacts to the same version within the project and by the way to the last version supported by the Apache Maven project see https://maven.apache.org/docs/history.html

Reply:
ok, please use `maven-compat.version` instead

I know, i know ... old version, not supported ... and I agree, but this change is irrelevant here (and part of another, inprogess, pull request)

Reply:
This is generating some security scans errors.

Which another PR is related to this?

Reply:
not public yet

Reply:
https://github.com/jenkinsci/pipeline-maven-plugin/pull/705